### PR TITLE
Ccg el #1205719 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://bugzilla.suse.com/show_bug.cgi?id=1205719#c4
https://github.com/SUSE/spacewalk/issues/20187

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
[uyuni_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/10433303/uyuni_client-configuration_guide.pdf)
see p. 73 s.

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 1]
```diff
@@ -150,6 +150,35 @@ The channels you need for this procedure are:
...
+
+| OS Version
+| Base Channel
+| Client Channel
+| Tools Channel
+
+| {sll} 7
+| rhel-x86_64-server-7
+| -
+| el7-uyuni-client-x86_64
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-sleses.adoc:168]:
```suggestion
| sll7-uyuni-client-x86_64
```

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 2]
```diff
@@ -150,6 +150,35 @@ The channels you need for this procedure are:
...
+
+| {sll} 8
+| rhel8-pool-x86_64
+| -
+| el8-uyuni-client-x86_64
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-sleses.adoc:173]:
```suggestion
| sll8-uyuni-client-x86_64
```

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 3]
```diff
@@ -150,6 +150,35 @@ The channels you need for this procedure are:
...
+
+| {sll} 9
+| el9-pool-x86_64
+| -
+| el9-uyuni-client-x86_64
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-sleses.adoc:178]:
```suggestion
| sll9-uyuni-client-x86_64
```

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 4]
```diff
@@ -150,6 +150,35 @@ The channels you need for this procedure are:
...
+.ES Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Tools Channel
+
+| {sll} 7
+| rhel-x86_64-server-7
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-sleses.adoc:166]: I wonder if it is`rhel-x86_64-server-7` or `res7-x86_64`.

See:

image

Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-sleses.adoc:166]: BTW, maybe this question is valid for SUSE Manager as well.

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 5]
```diff
@@ -119,10 +119,10 @@ include::snippets/addchannels_vendor_webui.adoc[]
 
 endif::[]
 
-
-ifeval::[{uyuni-content} == true]
 The channels you need for this procedure are:
 
+ifeval::[{uyuni-suma} == true]
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-sleses.adoc:124]: Isn't this supposed to be `suma-content`?

Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc:124]:
```suggestion
ifeval::[{suma-content} == true]
```

Comment (keichwa) [reply modules/client-configuration/pages/clients-sleses.adoc:124]: thanks for spotting!

File: modules/client-configuration/pages/clients-sleses.adoc (hunk 4 above)
Comment (keichwa) [reply modules/client-configuration/pages/clients-sleses.adoc:166]: I also could not find it yesterday.  @lucidd can you help?

**Comment by juliogonzalez** [comment]
I am not sure what to do about this PR.

I'd merge it as it is, and we can later adjust it for Uyuni, when we re-enable SLL

File: modules/client-configuration/pages/clients-sleses.adoc (hunk 4 above)
Comment (lucidd) [reply modules/client-configuration/pages/clients-sleses.adoc:166]: `"rhel-x86_64-server-7` is the base channel label. `res7-x86_64` is the updates channel if i see that correctly.

**Comment by keichwa** [comment]
> I am not sure what to do about this PR.
> 
> I'd merge it as it is, and we can later adjust it for Uyuni, when we re-enable SLL

I think we now can close https://bugzilla.suse.com/show_bug.cgi?id=1205719 --or is there sth obvious not covered?

**Comment by keichwa** [comment]
> I am not sure what to do about this PR.
> 
> I'd merge it as it is, and we can later adjust it for Uyuni, when we re-enable SLL

Thinking about it again, I agree.  @0rnela and @jcayouette can you please evaluate it?

File: modules/client-configuration/pages/clients-sleses.adoc (hunk 4 above)
Comment (keichwa) [reply modules/client-configuration/pages/clients-sleses.adoc:151]: we currently mention `rhel-x86_64-server-7` correctly as the base channel in the docs.

**Review by 0rnela — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]
LGTM. Thanks Karl.

**Pull request merged**